system/cache/providers/CFProvider.cfc

```diff
@@ -303,8 +303,9 @@ component
 	 * @objectKey The key to retrieve
      */
     function getQuiet( required objectKey ){
+		var ACFModernEngines = [ 2016, 2018, 2021 ];
```

Comment:
2016 shouldn't be added here in all reality, only 2018 plus. 
Also, why not just do a `gt 2018` for the version?